packages/apps/escrow-dashboard/src/components/Token/TokenView.tsx

```diff
@@ -1,14 +1,11 @@
-import { Grid, Typography } from '@mui/material';
+import { Grid } from '@mui/material';
 import * as React from 'react';
 import { CardTextBlock } from 'src/components/Cards';
-import BitfinexIcon from 'src/components/Icons/BitfinexIcon';
```

Comment:
Why has this been modified?

Reply:
I got an error from Lint because this was not used and forced me to delete to pass the tests